### PR TITLE
Use CSS variables over registerThemingParticipant - minimap viewparts

### PR DESCRIPTION
**Pull request by ghost**
#165169

### CODE REVIEW

**Comment by najmiehsa** [comment]
@microsoft-github-policy-service agree

File: src/vs/editor/browser/viewParts/minimap/minimap.css [hunk 1]
```diff
@@ -15,7 +15,18 @@
 	opacity: 1;
 }
 /* END cover the case that slider is visible on mouseover */
-
+.monaco-editor .minimap-slider .minimap-slider-horizontal {
+	color: var(--vscode-minimapSlider-background) !important;
```
Comment (rzhao271) [inline src/vs/editor/browser/viewParts/minimap/minimap.css:19]: Do these colours need to have `!important`, considering the original code doesn't use them?

Comment (najmiehsa) [reply src/vs/editor/browser/viewParts/minimap/minimap.css:19]: @rzhao271 Thank you for the review. 
I edit this.

**Review by rzhao271 — APPROVED** [review]

**Review by lramos15 — APPROVED** [review]

**Comment by alexdima** [comment]
@lramos15 @rzhao271 This introduced a regression. See https://github.com/microsoft/vscode/pull/166691

**Pull request merged**